hack/csv-generate.sh

```diff
@@ -94,17 +85,14 @@ if [[ "$CSV_VERSION" != "$PREV" ]] && [[ "$CSV_VERSION" != "$OLD" ]]; then
     "${MAINTAINERS}" \
     "${EXTRA_ANNOTATIONS}"
 
-    # restore the deleted CRD
-    cp "${CSV_DIR}/performance.openshift.io_performanceprofiles_crd.yaml" \
-      "${OUT_CSV_DIR}/performance.openshift.io_performanceprofiles_crd.yaml"
-fi
```

Comment:
fi deleted?

Reply:
you can found `f` here https://github.com/openshift-kni/performance-addon-operators/pull/428/files#diff-abe5157e9fb248d431200c7a3e6a18a9507be8497dd209f93569411139fb3952R98

Reply:
aha